### PR TITLE
Batch script to open replays directly

### PR DESCRIPTION
**Pull request by Chematronix**
Got tired of fiddling with launch options, so made this script.  Just copy it to "Documents\My Games\Company of Heroes 3\playback\" and drag & drop a .rec file to open it directly.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,12 +2,9 @@
...
 1. Extract the contents of the latest [release](https://github.com/Janne252/coh3-replay-enhancements/releases) `replay-enhancements.zip` to the Company of Heroes 3 install directory
     > Make sure that the path to the `init.scar` file is `<..>\steamapps\common\Company of Heroes 3\replay-enhancements\init.scar`
-1. Set game launch options
-    ```
-    -dev -replay playback:<replay>.rec
-    ```
+1. Copy [CoH3RecReplay.bat](https://github.com/Janne252/coh3-replay-enhancements/blob/master/CoH3RecReplay.bat) from the release to your replay folder (e.g. `Documents\My Games\Company of Heroes 3\playback\`) for easy access.
     > See https://cohdb.com/info for more detailed instructions of how the game launch options work
-1. Load up a replay by launching the game
+1. Drag & drop a .rec file onto `CoH3RecReplay.bat` to open the replay.
```
Comment (Janne252) [inline readme.md]: Could we keep the manual instructions available, something like

...
1. Load up a replay by 
    - Dragging a `.rec` file onto  [`CoH3RecReplay.bat`](https://github.com/Janne252/coh3-replay-enhancements/blob/master/CoH3RecReplay.bat)
    > Copy [`CoH3RecReplay.bat`](https://github.com/Janne252/coh3-replay-enhancements/blob/master/CoH3RecReplay.bat) to your replay folder (e.g. `Documents\My Games\Company of Heroes 3\playback\`) for easy access.
    - Or by manually setting the game's launch options to `-dev -replay playback:<replay>.rec`
    > See https://cohdb.com/info for more detailed instructions.

...

File: CoH3RecReplay.bat [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@Echo off
+For /F "Tokens=1,2*"  %%a In ('reg query HKCU\Software\Valve\Steam /v SteamExe') Do Set steamExe=%%c
+If Not [%~x1]==[.rec] ( 
+	Echo Copy this bat file to "Documents\My Games\Company of Heroes 3\playback\" and drag a CoH3 .rec file onto it to replay it
+	Pause
+) Else (
+	Echo Launching %steamExe% -applaunch 1677280 -dev -replay playback:"%~n1%~x1"
+	%steamExe% -applaunch 1677280 -dev -replay playback:"%~n1%~x1"
+	Timeout /T 30
+)
```
Comment (Janne252) [inline CoH3RecReplay.bat]: Could this file have a more descriptive name, e.g. `LaunchCoH3Replay.bat`?
I think it should also be included in releases. Could you add it to `build.bat`, please?

```bat
SET OUTPUT_FILENAME=".\replay-enhancements.zip"
CALL DEL %OUTPUT_FILENAME%
.bin\zip\7za.exe u %OUTPUT_FILENAME% LICENSE.txt README.md LaunchCoH3Replay.bat -ir!*.scar -xr!types -x!tests

```

Lastly, I couldn't get the batch script to run. I'm testing it on Windows 10. I'm not sure if the batch syntax you used is newer, perhaps Windows 11?

This works on my Windows 10:
```suggestion
@Echo off
For /F "Tokens=1,2*"  %%a In ('reg query HKCU\Software\Valve\Steam /v SteamExe') Do Set steamExe=%%c
If Not "%~x1" == ".rec" ( 
	Echo Copy this bat file to "Documents\My Games\Company of Heroes 3\playback\" and drag a CoH3 .rec file onto it to replay it
	Pause
) Else (
	Echo "Launching %steamExe% -applaunch 1677280 -dev -replay playback:%~n1%~x1"
	"%steamExe%" -applaunch 1677280 -dev -replay playback:"%~n1%~x1"
	Timeout /T 30
)
```

**Review by Janne252 — CHANGES_REQUESTED** [review]
This seems like a useful addition, thanks! 💪

File: CoH3RecReplay.bat (hunk 2 above)
Comment (Chematronix) [reply CoH3RecReplay.bat]: Done and done.  Thanks for doing Relic's work!

**Review by Janne252 — APPROVED** [review]

**Pull request merged**